themes/hugo-rladies/layouts/events/single.html

```diff
@@ -1,335 +1,164 @@
+      <div class="card-body">
+        <h2 class="card-title">{{ .n_events_past }}</h2>
+      </div>
+    </div>
     {{ end }}
-    </center>
-    <div id="menu" class = "mt-4 mb-2 pr-4">
-        <span class="dropdown">
-            <button id="dropdownMenu-calendarType" class="btn btn-primary btn-sm dropdown-toggle" type="button" data-toggle="dropdown"
-                aria-haspopup="true" aria-expanded="true">
-                <i id="calendarTypeIcon" class="calendar-icon ic_view_month" style="margin-right: 4px;"></i>
-                <span id="calendarTypeName">Dropdown</span>&nbsp;
-                <i class="calendar-icon tui-full-calendar-dropdown-arrow"></i>
-            </button>
-            <ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu-calendarType">
-                <li role="presentation">
-                    <a class="dropdown-menu-title" role="menuitem" data-action="toggle-daily">
-                        <i class="calendar-icon ic_view_day"></i>Daily
-                    </a>
-                </li>
-                <li role="presentation">
-                    <a class="dropdown-menu-title" role="menuitem" data-action="toggle-weekly">
-                        <i class="calendar-icon ic_view_week"></i>Weekly
-                    </a>
-                </li>
-                <li role="presentation">
-                    <a class="dropdown-menu-title" role="menuitem" data-action="toggle-monthly">
-                        <i class="calendar-icon ic_view_month"></i>Month
-                    </a>
-                </li>
-                <li role="presentation">
-                    <a class="dropdown-menu-title" role="menuitem" data-action="toggle-weeks2">
-                        <i class="calendar-icon ic_view_week"></i>2 weeks
-                    </a>
-                </li>
-                <li role="presentation">
-                    <a class="dropdown-menu-title" role="menuitem" data-action="toggle-weeks3">
-                        <i class="calendar-icon ic_view_week"></i>3 weeks
-                    </a>
-                </li>
-            </ul>
-        </span>
-        <span id="menu-navi">
-            <button type="button" class="btn btn-primary btn-sm move-today" data-action="move-today">Today</button>
-            <button type="button" class="btn btn-primary btn-sm move-day" data-action="move-prev">
-                <i class="fa fa-chevron-circle-left ic-arrow-line-left" data-action="move-prev"></i>
-            </button>
-            <button type="button" class="btn btn-primary btn-sm move-day" data-action="move-next">
-                <i class="fa fa-chevron-circle-right ic-arrow-line-right" data-action="move-next"></i>
-            </button>
-        </span>
-        <span id="renderRange" class="render-range"></span>
+  </center>
+  <div id="top">
+    <div style="float: left">
+      Timezone:
+      <select id="time-zone-selector">
+        <option value="America/Los_Angeles">
```

Comment:
We could technically render this list from an existing json of timezones, right? So we could generate the timezones based on that, rather than hard-code each TZ here?

Reply:
Trying to understand. Are we wanting all the timezones in the dropdown from a json: https://fullcalendar.io/demo-timezones.json, or just the six timezone in a separate json?

Reply:
https://github.com/rladies/website/pull/108/commits/9a98d4d20e3f414ca16e855359875c91a04c0b30 should add all timezones straight from a JSON file. If we don't want all, we can select the ones we need later.